### PR TITLE
Auth user profile

### PR DESCRIPTION
**Pull request by ragini07**
Added auth and user profile screens

https://user-images.githubusercontent.com/40401373/153707531-af3a2c76-c6ca-4ecd-95d2-d8a34dfe9899.mp4

### CODE REVIEW

File: Auth/Auth.css [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    text-align: center;
+    padding: 30px;
+}
+span{
+    line-height: 20px;
+    font-size: 14px;
+    margin:10px;
+}
+
+
```
Comment (pavan6115) [inline Auth/Auth.css:40]: Remove the comments

**Review by pavan6115 — APPROVED** [review]

File: Auth/Auth.css [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+/* a{
+    color:black;
+    text-decoration: none;
+    font-weight: bold;
+} */
```
Comment (pavan6115) [inline Auth/Auth.css]: Remove the comments

**Review by shraddha15-creator — APPROVED** [review]
Looks good

**Review by pavan6115 — APPROVED** [review]
Just remove the comments, rest is good 👍

**Review by ayush65 — APPROVED** [review]

File: Auth/Auth.css (hunk 2 above)
Comment (ragini07) [reply Auth/Auth.css]: done

**Pull request merged**